app/src/main/res/values/strings.xml

```diff
@@ -810,16 +810,16 @@
     <string name="security">Security</string>
     <string name="set_mode_for_app_ops_dots">Set mode for app ops…</string>
     <string name="backup_source_description">Back up the entire source directory.</string>
-    <string name="backup_apk_only_description">Back up only the APK files from the source directory. <b>Source</b> must be selected for this to work.</string>
-    <string name="backup_data_description">Back up data folders. If selected, internal data folders will be backed up by default.</string>
-    <string name="backup_external_data_description">Back up external data folders. <b>Data</b> must be selected for this to work.</string>
+    <string name="backup_apk_only_description">Only back up the APK files from the source directory you select.</string>
+    <string name="backup_data_description">Backs up data folders, optionally internal data folders too.</string>
```

Comment:
```suggestion
    <string name="backup_data_description">Backs up data folders, and optionally internal data folders too.</string>
```

Reply:
No, it's not that either. This flag means something like this: _Allow backing up data folders. If allowed, internal data folders are backed up by default._ For external data folders, they have to use the other option which won't work if this option isn't checked.